src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -188,7 +188,8 @@ public function findByIds($ids)
             ->addSelect('faxType')
             ->addSelect('notes')
             ->addSelect('medias')
-            ->where('account.id IN (:accountIds)');
+            ->where('account.id IN (:accountIds)')
+            ->orderBy('account.id', 'ASC');
```

Comment:
What impact does this have in terms of performance? I think we already had it like this once, but removed it because of performance issues. We also said that the order should not be relevant, so we started to change the tests this way (use `assertContains` instead of `assertEquals`) instead of fixing it this way. @chirimoya correct me if I am wrong.